### ISSUE TITLE
There is no need to add the explicit "expose" in the docker-compose file

### ISSUE THREAD

**Issue by mtheoryx** [MichaelJolley/io#46]
An example from the admin service in docker-compose.yml:

```
    ports:
      - "5090:80"
    expose:
      - "5090"
```

The container itself exposes the correct port. And the ports mapping handles the host:container mapping. There is no need to have an "expose" in docker-compose.yml

**Comment by MichaelJolley** [comment]
I dig it.  So just having:

```
ports: 
  - "5090:80"
```
will expose `5090` because it explicitly is mapping to the hosts port 5090.  That makes total sense.

**Comment by mtheoryx** [comment]
Closed by #57 

![bitmoji](https://render.bitstrips.com/v2/cpanel/7e6af720-36b5-4740-bd15-f8a9e26af9a2-c3b45c6e-2943-4fd5-ab6e-fe23bffcb90b-v1.png?transparent=1&palette=1&width=246)
